druntime/src/importc.h

```diff
@@ -96,3 +96,7 @@
 #if linux  // Microsoft won't allow the following macro
 #define __PRETTY_FUNCTION__ __func__
 #endif
+
+#if __APPLE__
```

Comment:
I assume DMD doesn't support 128 bit integers on any platforms?

Reply:
right.

Reply:
Then why are undefing it just for Apple?

Reply:
Because that's the only platform that complains about it.

Note that the Standard .h files are quite different on each platform.